search/tests.py

```diff
@@ -122,3 +122,17 @@ def test_search_page_num_queries(self, perform_solr_query):
         # Check that we perform one single query to get all sounds' information and don't do one extra query per sound
         with self.assertNumQueries(1):
             self.client.get(reverse('sounds-search'))
+
+    @mock.patch('search.views.perform_solr_query')
+    def test_search_page_with_filters(self, perform_solr_query):
+        perform_solr_query.return_value = self.perform_solr_query_response
+
+        # 200 response on sound search page access
+        resp = self.client.get(reverse('sounds-search'), {"f": 'grouping_pack:"Clutter" tag:"acoustic-guitar"'})
```

Comment:
If I understand correctly, this pack filter is wrong because it lacks the `ID_` part right? (it should be `grouping_pack:"1234_Clutter"`). So you're testing that pack filter is not applied right?

If that is the case I think you should add also an example where the filter is correct and check that it is there, and improve the comment so it is clear why the "bad" example is a bad example (why the pack filter is wrong).